CRM/Member/Form/MembershipRenewal.php

```diff
@@ -670,23 +667,19 @@ protected function sendReceipt($membership) {
     }
     CRM_Core_BAO_UFGroup::getValues($this->_contactID, $customFields, $customValues, FALSE, $members);
 
-    $this->assign('formValues', $this->_params);
+    $membership = Membership::get(FALSE)
```

Comment:
@mattwire I think we can ditch this entirely - see https://github.com/civicrm/civicrm-core/pull/30114